### PR TITLE
Tacobtracker

### PR DESCRIPTION
**Pull request by imazizahurt**

### CODE REVIEW

File: LoggingKata.Test/TacoParserTests.cs [hunk 1]
```diff
@@ -3,27 +3,44 @@
...
     {
-        [Fact]
-        public void ShouldDoSomething()
-        {
-            // TODO: Complete Something, if anything
-        }
-
         [Theory]
-        [InlineData("Example")]
+        [InlineData("33.168176,-87.521815,Taco Bell Tuscaloosa")]
```
Comment (chipelmer) [inline LoggingKata.Test/TacoParserTests.cs:9]: This is OK, but consider adding a few more test cases that perhaps push the boundaries of what should parse

File: LoggingKata.Test/TacoParserTests.cs [hunk 2]
```diff
@@ -3,27 +3,44 @@
...
         {
-            // TODO: Complete Should Parse
+            //Arrange
+            TacoParser tacoParser = new TacoParser();
+
+            // Act
+            ITrackable actual = tacoParser.Parse(str);
+
+            // Assert
+            Assert.NotNull(actual);
```
Comment (chipelmer) [inline LoggingKata.Test/TacoParserTests.cs:19]: Is there anything else that could be asserted to verify that the parsing was successful?

File: LoggingKata/Program.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 namespace LoggingKata
 {
-    class Program
+    class Progra
```
Comment (chipelmer) [inline LoggingKata/Program.cs:8]: Oops

File: LoggingKata/Program.cs [hunk 4]
```diff
@@ -16,14 +16,39 @@ static void Main(string[] args)
...
-            logger.LogInfo($"Lines: {lines[0]}");
+            //logger.LogInfo($"Lines: {lines[0]}");
 
             var parser = new TacoParser();
 
             var locations = lines.Select(parser.Parse).ToArray();
 
-            // TODO:  Find the two Taco Bells in Alabama that are the furthest from one another.
-            // HINT:  You'll need two nested forloops
+            Tacobell tacoBell = new Tacobell();
```
Comment (chipelmer) [inline LoggingKata/Program.cs:25]: This doesn't appear to be used and should be removed

File: LoggingKata/TacoParser.cs [hunk 5]
```diff
@@ -9,10 +9,40 @@ public class TacoParser
...
+            {
+                logger.LogError("Not enough info on line.");
+                return null;
+            }
+            else if (!double.TryParse(cells[0], out lat) || !double.TryParse(cells[1], out longit) || cells[2].Length == 0)
+            {
+                logger.LogError("Invalid data.");
+                return null;
+            }
+
```
Comment (chipelmer) [inline LoggingKata/TacoParser.cs:27]: This empty line separating the else-if should be removed

File: LoggingKata/TacoParser.cs [hunk 6]
```diff
@@ -9,10 +9,40 @@ public class TacoParser
...
+            else if (lat > 90)
```
Comment (chipelmer) [inline LoggingKata/TacoParser.cs:28]: What if `lat` is -91?

File: LoggingKata/TacoParser.cs [hunk 7]
```diff
@@ -9,10 +9,40 @@ public class TacoParser
...
+            {
+                logger.LogInfo("Latitude cannot be greater than 90.");
+                return null;
+            }
+            else if (longit > 180)
```
Comment (chipelmer) [inline LoggingKata/TacoParser.cs:33]: What if `longit` is -181?

**Review by chipelmer — APPROVED** [review]
Good work
